tests/test_data/users/complex_csv_a_b.user

```diff
@@ -0,0 +1 @@
+bhttps://githubb.combabcömplexbabсЮЛОажс$4vz*VçàhxpfCbbmwob

```

Comment:
This one made me laugh :smile: but I think we don't need it.
Better cover the `,` and `'` use-case instead.
If you remove this it's also fine since the rest of the tests are quite extensive already.